include/avif/avif.h

```diff
@@ -311,15 +311,17 @@ enum
     // This is actually reserved, but libavif uses it as a sentinel value.
     AVIF_COLOR_PRIMARIES_UNKNOWN = 0,
 
-    AVIF_COLOR_PRIMARIES_BT709 = 1, // Same as sRGB
+    AVIF_COLOR_PRIMARIES_SRGB = 1,
+    AVIF_COLOR_PRIMARIES_BT709 = 1,
     AVIF_COLOR_PRIMARIES_IEC61966_2_4 = 1,
     AVIF_COLOR_PRIMARIES_UNSPECIFIED = 2,
     AVIF_COLOR_PRIMARIES_BT470M = 4,
     AVIF_COLOR_PRIMARIES_BT470BG = 5,
     AVIF_COLOR_PRIMARIES_BT601 = 6,
     AVIF_COLOR_PRIMARIES_SMPTE240 = 7,
     AVIF_COLOR_PRIMARIES_GENERIC_FILM = 8,
-    AVIF_COLOR_PRIMARIES_BT2020 = 9, // Same as BT.2100
+    AVIF_COLOR_PRIMARIES_BT2020 = 9,
+    AVIF_COLOR_PRIMARIES_BT2100 = 9,
     AVIF_COLOR_PRIMARIES_XYZ = 10,
     AVIF_COLOR_PRIMARIES_SMPTE431 = 11,
     AVIF_COLOR_PRIMARIES_SMPTE432 = 12, // DCI P3
```

Comment:
optional: this is another candidate for an alias ("P3" or "DCI-P3" is a much more common name for it)